scipy/optimize/zeros.py

```diff
@@ -62,9 +67,22 @@ def results_c(full_output, r):
         return r
 
 
+def _results_select(full_output, r):
+    r"""Select from a tuple of (root, funccalls, iterations, flag)"""
+    x, funcalls, iterations, flag = r
+    if full_output:
+        results = RootResults(root=x,
+                              iterations=iterations,
+                              function_calls=funcalls,
+                              flag=flag)
+        return x, results
+    return x
+
+
 # Newton-Raphson method
 def newton(func, x0, fprime=None, args=(), tol=1.48e-8, maxiter=50,
-           fprime2=None):
+           fprime2=None,
+           full_output=False, disp=True):
```

Comment:
New keyword arguments need to be added last (otherwise they can break backward compat)

Reply:
Aren't the new keywords `full_output` and `disp` added last?  `fprime2` was the last argument, and `full_output` and `disp` appear after `fprime2`.

Reply:
No problem, thanks for looking at it.